glusterd2/middleware/auth_test.go

```diff
@@ -29,19 +31,23 @@ func TestGetAuthSecret(t *testing.T) {
 	assert.NotNil(t, secret)
 }
 
-func getAuthToken(username string, password string) string {
-	// Create the Claims
-	claims := &jwt.StandardClaims{
-		ExpiresAt: time.Now().Add(time.Second * time.Duration(120)).Unix(),
-		Issuer:    username,
-	}
-	token := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
-	ss, err := token.SignedString([]byte(password))
+func getAuthToken(username string, password string, r *http.Request) {
+	// Generate qsh
+	qshstring := "GET&/"
+	hash := sha256.New()
+	hash.Write([]byte(qshstring))
+	// Create Token
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+		"iss": username,
+		"exp": time.Now().Add(time.Second * time.Duration(120)).Unix(),
+		"qsh": hex.EncodeToString(hash.Sum(nil)),
```

Comment:
Can't use use `utils.GenerateQsh` here too? I know this is a test, and maybe you need some customization. If you need some customization, I'd just like to know why.

Reply:
i tried to use ```utils.GenerateQsh``` here for the testing, later i came to know that i cant use it  because of the ```testserver```.